### PR TITLE
Profile style

### LINKED ISSUE #9: Component: Form
As a user, 
When I navigate to "Find a Roommate", 
I should be able to complete and submit a dropdown form,
so that I can choose a semester to participate in the program.

### LINKED ISSUE #15: Profile-component
As a user,
* I would like to view resident profiles to view how well they fit my abilities.

### LINKED ISSUE #18: Fix Bug: Form/FindARoommate
As a user, 
when I select a semester, 
the thumbnails grid should populate with available residents on first selection rather than 2nd,
so that I know that the form is working correctly,
and residents are available for the chosen semester.

### PR DESCRIPTION
**Pull request by lo-la-do-li**
### What’s this PR do?
**Features:**
- Profile now renders on Thumbnail/ profile icon click, and closes on 'X' button click
- Profile view conditional renders full screen, centered, and hides Form and Thumbnails
- Apply for Housing button added to profile
- SilverNest logo added to NavBar

**Fixes:**
- Available residents display on first selection of semester in Form (refactored useEffect in FindARoommate)
- Resident Profile now closes on 'x' click and returns to thumbnail/form display
### Where should the reviewer start?
FindARoommate.js: 38
Thumbnails.js: 64
Profile.js: 59

### How should this be manually tested?
- npm start, navigate to FindARoommate, select a semester
- see available residents populate correctly first time
- click on thumbnail
- see resident profile expand and center
- click 'X' to close and return to thumbnails
### Any background context you want to provide?
n/a
### Screenshots (if appropriate):
Screen Shot 2021-01-12 at 11 25 56 AM
Screen Shot 2021-01-12 at 11 26 17 AM

### Questions:
If we have time to get gridlist tiles to hover, should we delete the profile icon from the thumbnails that is currently being used to indicate profile view?

### CODE REVIEW

File: src/FindARoommate/FindARoommate.js [hunk 1]
```diff
@@ -24,27 +30,24 @@ export default function FindARoomate() {
...
 
   useEffect(() => {
-    if (!availableResidents.length) {
-      getResidents();
-    } else {
-      getAvailableResidents(semester);
-    }
-  }, [])
+  
+  }, [semester, availableResidents])
```
Comment (cameronRomo) [inline src/FindARoommate/FindARoommate.js:40]: So clever! I'll have to put this one in my back pocket.

Comment (lo-la-do-li) [reply src/FindARoommate/FindARoommate.js:40]: The power of useEffect hook!

File: src/Profile/Profile.js [hunk 2]
```diff
@@ -34,19 +37,30 @@ const useStyles = makeStyles((theme) => ({
...
+function ResidentCard({resident, exitProfileView}) {
   const classes = useStyles();
   const [expanded, setExpanded] = React.useState(false);
+  // const [close, setClose] = React.useState(false)
 
   const handleExpandClick = () => {
     setExpanded(!expanded);
   };
 
+  const handleCloseClick = () => {
```
Comment (cameronRomo) [inline src/Profile/Profile.js:59]: This is way more adept than just refreshing the page.

**Review by cameronRomo — APPROVED** [review]
This looks great, I'm glad you were able to figure out how to enable the 'x' button.

File: src/FindARoommate/FindARoommate.js (hunk 1 above)
Comment (chadrick-d-dev) [reply src/FindARoommate/FindARoommate.js:40]: Super powerful!

File: src/Profile/Profile.js (hunk 2 above)
Comment (chadrick-d-dev) [reply src/Profile/Profile.js:59]: I'd like to review this when we have a moment.

**Review by chadrick-d-dev — APPROVED** [review]

**Pull request merged**